### ISSUE TITLE
Error in CMS with LeftAndMain::extra_requirements_css

### ISSUE THREAD

**Issue by phptek** [silverstripe/silverstripe-admin#334]
PHP 5.6.30 / framework + CMS 4.0.0-rc1 / silverstripe/admin 1.0.0-rc1

Attempting to add admin-specific CSS using the following config as per `LeftAndMain.php:232`. 

```
SilverStripe\Admin\LeftAndMain:
  extra_requirements_css:
    - app/css/socials.css:
         # Note indentation with 2 or 3 spaces results in the same problem
         media: screen
```
Results in the following error displayed within the CMS in all three panes (dev mode):

```
[Warning] preg_match() expects parameter 2 to be string, array given
GET /admin/settings?flushtoken=d17ad2d39cf4bad6cd9472c75f55f830&flush=1

Line 99 in /var/www/html/myproject/vendor/silverstripe/framework/src/Core/Manifest/ModuleResourceLoader.php
```

Removing the media type declaration, prevents the problem. The syntax looks funky to me anyway, so suggest the docblock is updated to reflect the true way this should be implemented in userland, or just deleted.

**Comment by flamerohr** [comment]
I think this may be a syntax issue, I've tried this out fixing what I thought were the syntax issues and it worked fine:
```
SilverStripe\Admin\LeftAndMain:
  extra_requirements_css:
    mysite/css/dummy.css:
      # Note indentation with 2 or 3 spaces results in the same problem
      media: screen
```

**Comment by flamerohr** [comment]
Feel free to re-open the issue if it's still an issue @phptek :)

**Comment by phptek** [comment]
@flamerohr  it's simply the docblock that is erroneous and needs fixing:

```
 226     /**
 227      * YAML configuration example:
 228      * <code>
 229      * LeftAndMain:
 230      *   extra_requirements_css:
 231      *     - mysite/css/mystyle.css:
 232      *          media: screen
```

**Comment by dhensby** [comment]
fix at #336
